### PR TITLE
sru: 20.4.0 openstack ipv6 validation

### PR DESCRIPTION
**Pull request by blackboxsw**

### CODE REVIEW

**Review by lucasmoura — COMMENTED** [review]
Just to confirm as well, is it expected to not have logs for `groovy` ?

File: 20201201/README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The links listed below are to bugs fixed in this SRU and the verification of tho
 | [GH: #632](https://github.com/canonical/cloud-init/commit/2730521fd566f855863c5ed049a1df26abcd0770) | [verification output](../bugs/gh-632.txt) |
 | [GH: #660](https://github.com/canonical/cloud-init/commit/66b4be8b6da188a0667bd8c86a25155b6f4f3f6c) | [verification output](../bugs/gh-660.txt) |
 | [GH: #671](https://github.com/canonical/cloud-init/commit/eea754492f074e00b601cf77aa278e3623857c5a) | [verification output](../bugs/gh-671.txt) |
+| [GH: #634](https://github.com/canonical/cloud-init/commit/b46e4a8c) | [verification output](../bugs/gh-634-ipv6-openstack.txt) |
```
Comment (OddBloke) [inline 20201201/README.md:22]:
```suggestion
| [GH: #634](https://github.com/canonical/cloud-init/commit/b46e4a8c) | [verification output](../bugs/gh-634-ipv6-openstack.txt) |
| [GH: #660](https://github.com/canonical/cloud-init/commit/66b4be8b6da188a0667bd8c86a25155b6f4f3f6c) | [verification output](../bugs/gh-660.txt) |
| [GH: #671](https://github.com/canonical/cloud-init/commit/eea754492f074e00b601cf77aa278e3623857c5a) | [verification output](../bugs/gh-671.txt) |
```

**Review by OddBloke — COMMENTED** [review]
Same Q as Lucas: should we have groovy in here? Otherwise, looks good!

**Review by lucasmoura — APPROVED** [review]

**Review by OddBloke — APPROVED** [review]
Thanks!

**Pull request merged**